docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
```

Comment:
Not sure what you mean by "live at the root". All content should still be in the `templates` directory, although we might want to rename that directory to `content` as proposed in a comment on an earlier version of this file.